prow/plugins/override/override.go

```diff
@@ -186,6 +274,12 @@ func handle(oc overrideClient, log *logrus.Entry, e *github.GenericCommentEvent)
 		return oc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, user, resp))
 	}
 
+	if !authorized && !authorizedTopLevelOwner(oc, options.AllowTopLevelOwners, log, org, repo, user, pr) {
+		resp := fmt.Sprintf("%s unauthorized: /override is restricted to repo administrators or top level code owners", user)
```

Comment:
Could we please build up this error message using the provided options? Could re-use the logic to build it for the plugin help endpoint. I think it would be awesome to tell the users if, specifically, OWNERS is enabled, or which GitHub Teams are.

Reply:
Thank you for the review. I have updated the PR.